mountd/mountd.c

```diff
@@ -189,17 +190,18 @@ static int write_file_safe(char *path, char *buff, size_t length, int mode)
 	ret = 0;
 
 err_out:
+	g_free(buff);
```

Comment:
Is it better to make callers of this function free it?

Reply:
I'm not sure. it would involve turning

```
return x;
```

to

```
int i = x;
g_free(y);
return i;
```